src/Export/Field/CMS/CustomFields.php

```diff
@@ -0,0 +1,152 @@
+<?php
```

Comment:
There is a lot of code duplicated from `Export\Field\CustomFields.php`
![image](https://user-images.githubusercontent.com/4511883/128987800-45ea5c14-a315-4190-a312-09af956c8842.png)
definitely beneficial to abstract this code in one class